src/utils/node_byte_utils.ts

```diff
@@ -23,6 +23,25 @@ type NodeJsBufferConstructor = Omit<Uint8ArrayConstructor, 'from'> & {
 // Node.js global
 declare const Buffer: NodeJsBufferConstructor;
 
+/** @internal */
+export function mathRandomBytes(byteLength: number) {
```

Comment:
Why duplicate this function?  We could easily put this in the shared utils file:
```typescript
export function mathRandomBytes(byteUtils: ByteUtils, byteLength: number) {
  return byteUtils.fromNumberArray(
    Array.from({ length: byteLength }, () => Math.floor(Math.random() * 256))
  );
}
```

Reply:
This was to avoid a circular dependency on ByteUtils